repos/spack_repo/builtin/packages/pika/package.py

```diff
@@ -166,6 +166,8 @@ class Pika(CMakePackage, CudaPackage, ROCmPackage):
     with when("+stdexec"):
         depends_on("stdexec")
         depends_on("stdexec@24.09:", when="@0.29:")
+        # https://github.com/pika-org/pika/issues/1448
+        conflicts("stdexec@25.09:")
```

Comment:
```suggestion
        conflicts("stdexec@25.09:")
```
Worth adding a `msg`?

Reply:
Very good point, added one in https://github.com/spack/spack-packages/pull/1974/commits/c3be3b4fc263febd6f74bacdf4611f5264a36e32.

Reply:
Amazing, thanks.